pages/blog/index.vue

```diff
@@ -95,6 +96,12 @@
           return post.primaryImageOfPage.image.contentType;
         }
         return null;
+      },
+      imageAlt(post) {
+        if (post.primaryImageOfPage && post.primaryImageOfPage.image.description) {
```

Comment:
wouldn't `if (post.primaryImageOfPage.image.description)` suffice?

Reply:
If there is no primaryImageOfPage you'd get an error, something like "Can't read property image of undefined".